packages/console-app2/src/substrate-lib/components/TxButton.tsx

```diff
@@ -14,7 +15,7 @@ function TxButton ({
   accountPair = null,
   label,
   setStatus,
-  color = 'blue',
+  // color = 'blue',
```

Comment:
what's the purpose of this? is it leftover or intentional?

Reply:
Setting color will not compile, but since we probably want to do it later, I left it in form of a comment. OK?

Reply:
then pls add a word of comment to show your intent like `// color = 'blue', <- won't compile, but left for restyle later`

Reply:
OK, I added a TODO comment